### ISSUE TITLE
Spring boot generations GH action is failing

### ISSUE THREAD

**Issue by anthonydahanne** [paketo-buildpacks/spring-boot#320]
## Expected Behavior
The GH action defined here: https://github.com/paketo-buildpacks/spring-boot/blob/main/.github/workflows/update-spring-generations.yml should work and generate an updated https://github.com/paketo-buildpacks/spring-boot/blob/main/spring-generations.toml

## Current Behavior
it fails

## Possible Solution
The spring website team has changed the Spring Boot generations URL from https://spring.io/api to https://api.spring.io/projects/spring-boot/generations

The GH action code could need to be updated since it appends `/projects` to the uri that is given to it

## Alternate solution
It could be that a URL rewrite rule could be implemented, but it would just be a workaround I guess 

## Steps to Reproduce
You can look at a recent failure: https://github.com/paketo-buildpacks/spring-boot/actions/runs/4120288901

## Motivations
The scheduled action is down

**Comment by anthonydahanne** [comment]
actually it's pretty easy to fix

```
docker run -it -v /Users/myuser/output:/output   --env GITHUB_OUTPUT=output --env INPUT_URI=https://api.spring.io ghcr.io/paketo-buildpacks/actions/spring-generations:main
```

**Comment by anthonydahanne** [comment]
this issue can be closed, it's a duplicate of #319
